websites/N/Netflix/presence.ts

```diff
@@ -419,3 +420,19 @@ presence.on("UpdateData", async () => {
   else if (!showBrowsing) presence.setActivity();
   else presence.setActivity(presenceData);
 });
+
+const shortenedURLs: Record<string, string> = {};
+async function getShortURL(url: string) {
+  if (!url || url.length < 256) return url;
```

Comment:
The types aren't right if url is nullable

Reply:
Nah it shouldn't be, I just put it to be sure in the case it is. Since this functions should be used by others and for sure someone will put a null able in...